src/neo/IO/Helper.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Numerics;
```

Comment:
Does this change needs to come in this PR?
Maybe it is better to open a new one first and we can approve it beforehand.